src/Datadog.Trace.ClrProfiler.Managed.Loader/Startup.NetCore.cs

```diff
@@ -19,7 +19,13 @@ public partial class Startup
         private static string ResolveManagedProfilerDirectory()
         {
             string tracerFrameworkDirectory = "netstandard2.0";
-            var tracerHomeDirectory = Environment.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME") ?? string.Empty;
+
+            var tracerHomeDirectory = Environment.GetEnvironmentVariable("DD_DOTNET_TRACER_HOME");
+            if (string.IsNullOrWhiteSpace(tracerHomeDirectory))
```

Comment:
.NET Core: Fallback to `DD_INTEGRATIONS` if `DD_DOTNET_TRACER_HOME` was not set.